source/js/menu.bindings.js

```diff
@@ -7,13 +7,33 @@
 
   Drupal.behaviors.gentBaseLoadMenu = {
     attach: function (context, settings) {
-      if (!Menu) { // eslint-disable-line no-undef
+      if (!Modal) { // eslint-disable-line no-undef
         return;
       }
 
-      var selected = document.querySelectorAll('nav.menu');
-      for (var i = selected.length; i--;) {
-        new Menu(selected[i]); // eslint-disable-line no-undef
+      var menu = document.querySelectorAll('.modal.menu');
+
+      var createModal = function (modal) {
+        // eslint-disable-next-line no-undef
+        new Modal(modal, {
+          // The modal is always visible from tablet and up,
```

Comment:
Not sure if this is still true.
Isn't the menu duplicate now?

Reply:
As discussed, first change this in style guide if necessary, then this repo can update too...